close to final version please test

grantozz opened the pull request.

grantozz committed:
close to final version please test

WindowDump commented on src/BracketPane.java:
```diff
@@ -401,9 +408,11 @@ private void createVertices(int iX, int iY, int iXO, int iYO, int num, int incre
                                 }
                                 ArrayList<Integer> tmpHelp = helper(location, num);
                                 for (int j = 0; j < aNodeList.size(); j++) {
+                                        System.out.println(currentBracket.getBracket().get(tmpHelp.get(j)));
```
These print statements should be commented out.

WindowDump committed:
Commenting out print statements

Commenting out the print statements for a clean runtime.

WindowDump committed:
switch to scoreboard after simulation

After the simulate button is pressed, the simulation is run, and the brackets are scored, the scoreboard will then be displayed.
